src/os/bluestore/BlueFS.cc

```diff
@@ -53,7 +53,7 @@ BlueFS::~BlueFS()
 
 void BlueFS::_init_logger()
 {
-  PerfCountersBuilder b(cct, "BlueFS",
```

Comment:
Commit message says "log", but this is just adding a nick, meaning it will show up in the schema and be grabbed by daemonperf (what you refer to as 'surface' in the next commit), right?